src/test/scala/org/clulab/wm/eidos/text/eval6/TestDoc2.scala

```diff
@@ -1,27 +1,190 @@
+                 |hunger catastrophe continues to threaten parts of the country, three UN agencies warned
+                 |today.
+               """
     val tester = new Tester(text)
-  
-    val concerns = NodeSpec("Concerns about insufficient food access") // Make some node spece
-    
+
     behavior of "TestDoc2 Paragraph 1"
 
-    failingTest should "have correct edges 1" taggedAs(Somebody) in {
-      tester.test(EdgeSpec(concerns, Causal, concerns)) should be (successful) // Test edges connecting them
+    ignore should "have correct output #1" taggedAs(Somebody) in {
+      val food = NodeSpec("food", Dec("shortages", "severe"))
+      val hunger = NodeSpec("hunger", Dec("catastrophe"))
+
+      tester.test(food) should be (successful)
+      tester.test(hunger) should be (successful)
+    }
+  }
+
+  { // Paragraph 2
+    val text = """
+                 |The Food and Agriculture Organization of the United Nations (FAO), the United Nations
+                 |Children’s Fund (UNICEF) and the World Food Programme (WFP) stressed that while the
+                 |deteriorating situation coincides with an unusually long and harsh annual lean season, when families
+                 |have depleted their food stocks and new harvests are not expected until August, the level of food
+                 |insecurity this year is unprecedented.
+               """
+    val tester = new Tester(text)
+
+    behavior of "TestDoc2 Paragraph 2"
+
+    ignore should "have correct output #2" taggedAs(Somebody) in {
+      val situation = NodeSpec("situation", Dec("deteriorating"))
+      val leanSeason = NodeSpec("annual lean season", Quant("long", "unusually"), Quant("harsh"))
+      val foodStocks = NodeSpec("food stocks", Dec("depleted"))
+      // TODO: must decide where we handle transparent nouns such as "level"
+      val foodInsecurity = NodeSpec("level of food insecurity", Quant("unprecedented"))
+
+      tester.test(situation) should be (successful)
+      tester.test(leanSeason) should be (successful)
+      tester.test(foodStocks) should be (successful)
+      tester.test(foodInsecurity) should be (successful)
+      tester.test(EdgeSpec(situation, Correlation, leanSeason)) should be (successful)
+
+      // TODO: must test for the relation "families" => "food stocks" NOT being present!
+    }
+  }
+
+  { // Paragraph 3
+    val text = """
+                 |This is the highest level of hunger since the conflict in South
+                 |Sudan began two-and-a-half years ago.
+               """
+    val tester = new Tester(text)
+
+    behavior of "TestDoc2 Paragraph 3"
+
+    ignore should "have correct output #3" taggedAs(Somebody) in {
+      val hunger = NodeSpec("level of hunger", Quant("highest"))
+
+      tester.test(hunger) should be (successful)
+    }
+  }
+
+  { // Paragraph 4
+    val text = """
+                 |“We are very worried to see that food insecurity is spreading beyond conflict areas as rising prices,
```

Comment:
also a
`val insecurity = NodeSpec("food insecurity", Inc("spreading"))` ?
and... IDK, we really need to get these predicates, the thing being caused is really "families accessing food"... not just families, we need to talk about these Monday I think bc we need to add that functionality

Reply:
and if that -- the insecurity would be correlated with the other events... though we can't handle hyper edges yet